tests/integration/telemetry/stats/prometheus/footprint/footprint_test.go

```diff
@@ -0,0 +1,474 @@
+		"envoy_cluster_manager_cds_update_attempt":                                     true,
+		"envoy_cluster_manager_cds_update_duration_bucket":                             true,
+		"envoy_cluster_manager_cds_update_duration_count":                              true,
+		"envoy_cluster_manager_cds_update_duration_sum":                                true,
+		"envoy_cluster_manager_cds_update_failure":                                     true,
+		"envoy_cluster_manager_cds_update_rejected":                                    true,
+		"envoy_cluster_manager_cds_update_success":                                     true,
+		"envoy_cluster_manager_cds_update_time":                                        true,
+		"envoy_cluster_manager_cds_version":                                            true,
+		"envoy_cluster_manager_cluster_added":                                          true,
+		"envoy_cluster_manager_cluster_modified":                                       true,
+		"envoy_cluster_manager_cluster_removed":                                        true,
+		"envoy_cluster_manager_cluster_updated":                                        true,
+		"envoy_cluster_manager_cluster_updated_via_merge":                              true,
+		"envoy_cluster_manager_update_merge_cancelled":                                 true,
+		"envoy_cluster_manager_update_out_of_merge_window":                             true,
+		"envoy_cluster_manager_warming_clusters":                                       true,
+		"envoy_cluster_max_host_weight":                                                true,
+		"envoy_cluster_membership_change":                                              true,
+		"envoy_cluster_membership_degraded":                                            true,
+		"envoy_cluster_membership_excluded":                                            true,
+		"envoy_cluster_membership_healthy":                                             true,
+		"envoy_cluster_membership_total":                                               true,
+		"envoy_cluster_metadata_exchange_alpn_protocol_found":                          true,
+		"envoy_cluster_metadata_exchange_alpn_protocol_not_found":                      true,
+		"envoy_cluster_metadata_exchange_header_not_found":                             true,
+		"envoy_cluster_metadata_exchange_initial_header_not_found":                     true,
+		"envoy_cluster_metadata_exchange_metadata_added":                               true,
+		"envoy_cluster_original_dst_host_invalid":                                      true,
+		"envoy_cluster_retry_or_shadow_abandoned":                                      true,
+		"envoy_cluster_ssl_ciphers_ECDHE_RSA_AES256_GCM_SHA384":                        true,
+		"envoy_cluster_ssl_connection_error":                                           true,
+		"envoy_cluster_ssl_curves_P_256":                                               true,
```

Comment:
This will likely work in an isolated test environment but when ran against an Envoy using serving real traffic may differ